pkgs/applications/networking/p2p/fragments/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, fetchgit, meson, ninja, cmake, desktop-file-utils, automake, autoconf, libtool, libevent, openssl, zlib, pkgconfig, libgee, curl, vala, glib, python3, gtk3, libhandy, hicolor-icon-theme, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "fragments";
+  version = "1.3";
+
+  src = fetchgit {
```

Comment:
Please use `fetchFromGitLab` because
1. It allows us to fix GitLab specific issues
2. It might be more efficient

Reply:
`fetchFromGitLab` does not support submodules. Though, apparently, they are no longer used.